tests/email.test.js

```diff
@@ -0,0 +1,41 @@
+"use strict";
+
+const nodemailer = require("nodemailer");
+
+const EmailUtils = require("../email-utils");
+
+
+jest.mock("nodemailer");
+
+
+test("EmailUtils.init with empty host doesnt invoke nodemailer", () => {
+  nodemailer.createTransport = jest.fn();
+
+  EmailUtils.init("");
```

Comment:
`EmailUtils.init()` is async above, should we do an `await` here (and make parent func async as well)?

```js
async init(smtpUrl = AppConstants.SMTP_URL) {..}
```

Same w/ the 2 other calls below. Doesn't look like `EmailUtils.sendEmail()` is async, so I think that one is fine as-is.

Reply:
`init` is a bit strange and could use a refactor I think. It returns `Promise.resolve()` values, so even though it's marked as `async` so it can `await` the `gTransporter.verify()`, it returns final values.